packages/demo/src/components/examples/modalWindowExamples.tsx

```diff
@@ -58,17 +59,74 @@ const ModalExampleDisconnected: React.FunctionComponent<ModalConnectedExamplePro
         close(id);
         toggleIsDirty(id, false);
     };
-
+    const modalConfirmationId = 'Modal-Confirmation-Connected';
+    const modalWarningId = 'Modal-Warning-Connected';
+    const modalInfoId = 'Modal-Info-Connected';
     const modalId = 'Modal-Connected-#1';
     return (
         <Section level={2} title="Modals connected to the redux store">
+            <Section level={3} title="Confirmation, warning and info prompts">
+                <Button className="btn" onClick={() => open(modalConfirmationId)}>
+                    Prompt confirmation
+                </Button>
+
+                <ModalCompositeConnected
+                    id={modalConfirmationId}
+                    title="Prompt confirmation"
+                    isPrompt
+                    classes={'mod-prompt'}
```

Comment:
I'm surprised that we have a `isPrompt` prop and that we still need to specify the `mod-prompt` class :thinking:

Reply:
i tried it and it didn't change the modal.. :S